src/components/pages/coach/Anamnesis/index.jsx

```diff
@@ -157,14 +176,31 @@ class Anamnesis extends React.Component{
         }
     }
 
+    getBasicUserData = () => {
+        this.props.client.query({
+        query: GET_BASIC_USERDATA,
+        variables: { "token": localStorage.getItem('fprint'), "id": this.state.uid }
```

Comment:
"fprint" doesn't seem to be the best naming for an JWT in local storage.
I would prefer to name it anonymousJWT or something along those lines,
cause it's the JWT of the anonymous user.

Reply:
I have to agree on that one. They are a remenant of darker times. 🌋